examples/hana-scale-up-perf-optimized-azure.yaml

```diff
@@ -453,3 +453,20 @@ groups:
           ## References
           - section 2 in https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
         scored: true
+      - id: 2.2.9
+        description: "SAP Host Agent is installed"
+        audit: |
+          sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:upper:]' '[:lower:]')
+          inst=$(sudo crm configure show | grep -m1 InstanceNumber= | sed -e "s/.*InstanceNumber=\(..\).*/\1/")
+          sudo runuser -l ${sid}adm -c "sapcontrol -nr $inst -function CheckHostAgent"
```

Comment:
Now I understand why you are asking, if you can check from any <sid>adm user.

Reply:
An alternative would be (as root):
cd /usr/sap/hostctrl/exe
./saphostctrl -function Ping   (and/or)
./sapcontrol -nr 99 -function CheckHostAgent

In your case you might get false positives, if the sapstartsrv for the found instance is not available, because it is running on a different cluster node,

Reply:
@fmherschel Thank you for the advice, I have amended the PR!

Reply:
@fmherschel Thanks also for the feedback.
Regarding the potential false positives - isn't the Host Agent always local to each cluster node? If so I assume we would be safe from false positives for the Host Agent. 

I guess instance agents/sapstartsrv are the ones to be careful with?

Reply:
@lee-martin I am not sure, if I understand your comment. False-Positives are always bad. As well as False-Negatives. If we feel save with a planend false test, why do we implement it? And the code I requested a check was running against the instance sapstartsrv and not against the real saphostagent.